.github/workflows/sub-deploy-integration-tests-gcp.yml

```diff
@@ -134,7 +134,7 @@ jobs:
     runs-on: zfnd-runners
     needs: [ get-disk-name ]
     if: ${{ !cancelled() && !failure() }}
-    timeout-minutes: ${{ inputs.is_long_test && 7200 || 180 }}
+    timeout-minutes: ${{ inputs.is_long_test && 7200 || 600 }}
```

Comment:
We should rever this after the PR merges; with the parallelization it should be OK but I didn't want to possibly waste yet another CI run if it does timeout